#
# Copyright (c) 2015, 2016, Oracle and/or its affiliates. All rights reserved.
#
avoid memory leak in ucnvsel_open
http://bugs.icu-project.org/trac/ticket/12388
--- icu4c-73_1/icu/source/common/ucnvsel.cpp
+++ icu4c-73_1/icu/source/common/ucnvsel.cpp
@@ -219,6 +219,7 @@ generateSelectorData(newSelector.getAlia
   upvec_close(upvec);
 
   if (U_FAILURE(*status)) {
+    uprv_free(allStrings);
     return nullptr;
   }
 
